exercises/pangram/src/test/java/PangramsTest.java

```diff
@@ -50,20 +50,20 @@ public void missingLettersReplacedByNumbers() {
 
     @Ignore
     @Test
-    public void pangramWithMixedCaseAndPunctuation() {
+    public void pangramWithMixedCaseAndPunctuationIsRecognizedAsPangram() {
         assertTrue(Pangrams.isPangram("\"Five quacking Zephyrs jolt my wax bed.\""));
     }
 
     @Ignore
     @Test
-    public void pangramWithNonAsciiCharacters() {
+    public void pangramWithNonAsciiCharactersIsRecognizedAsPangram() {
         assertTrue(Pangrams.isPangram("Victor jagt zwölf Boxkämpfer quer über den großen Sylter Deich."));
     }
 
 
     @Ignore
     @Test
-    public void panagramInAlphabetOtherThanAscii() {
+    public void panagramInAlphabetOtherThanAsciiIsNotPangram() {
```

Comment:
Maybe `...IsNotRecognizedAsPangram` is better than `...IsNotPangram` since as far as I understand it this is a pangram just in another alphabet?